src/main/java/teammates/logic/api/FirebaseInstance.java

```diff
@@ -0,0 +1,53 @@
+package teammates.logic.api;
+
+import teammates.common.exception.FirebaseException;
+import teammates.common.util.Config;
+import teammates.logic.external.AuthService;
+import teammates.logic.external.EmptyAuthService;
+import teammates.logic.external.FirebaseService;
+
+/**
+ * Provides Firebase services.
+ */
+public class FirebaseInstance {
```

Comment:
This class is not Firebase instance. This is a proxy to an auth service, which may or may not be Firebase.